packages/uikit-workshop/.patternlabrc.example.js

```diff
@@ -2,4 +2,5 @@
 
 module.exports = {
   // buildDir: __dirname + '/www/pattern-lab',
+  noViewAll: true, // uncomment to disable displaying viewAll links in Pattern Lab
```

Comment:
this comment doesnt make sense, as it's enabled, not commented

Reply:
Derp — I think I was thinking we’d want this option to be disabled by default (hence the comment) but I probably I uncommented that when I was putting together my PR screenshot 🤦‍♂️

Reply:
bump